### DISCUSSION TITLE
Caching

### DISCUSSION THREAD

**Discussion by DevFreeze** [getsentry/sentry-dart#704] [Q&A] [answered]
Are there any additional options required to have the cached envelopes sent once the network comes back up?

Any cached envelopes are never delivered to Sentry, even if the network connection is restored.

**Comment by marandaneto** [comment, marked as answer]
@DevFreeze no, cached envelopes are gonna be sent on App restart only, if there's connection.

**Reply by DevFreeze** [reply]
Aaah, thanks a lot :D
